stock_picking_filter_lot/views/stock_picking_view.xml

```diff
@@ -9,7 +9,7 @@
         <field name="model">stock.picking</field>
         <field name="inherit_id" ref="stock.view_picking_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='move_line_ids']/tree/field[@name='lot_id']" position="attributes">
+            <xpath expr="//field[@name='move_line_ids_without_package']/tree/field[@name='lot_id']" position="attributes">
```

Comment:
why this change?

Reply:
@lreficent move_line_ids no longer exists this way in the record. It got changed to move_line_ids_without_package

Reply:
Oh, I see, you are right. Thanks.